src/main/java/com/volasoftware/tinder/constants/AccountConstant.java

```diff
@@ -2,11 +2,13 @@
 
 public class AccountConstant {
 
-    public static final String NOT_FOUND = "Account not found!";
-    public static final String VERIFIED = "Account verified!";
-    public static final String REGISTERED = "Account registered!";
-    public static final String LOGGED_IN = "Account logged in!";
-    public static final String WRONG_PASSWORD = "Wrong password";
-    public static final String NOT_VERIFIED = "Account not verified!";
-    public static final String DETAILS = "Account details!";
+  public static final String NOT_FOUND = "Account not found!";
```

Comment:
The code style is changed. What is the reason?

Reply:
I`m not exactly sure what is happening, but when my Intellij updates, my plugins, code style and annotation processing settings are gone, so I guess some changes were made without code style...